hazelcast/src/main/java/com/hazelcast/collection/impl/list/ListContainer.java

```diff
@@ -200,9 +200,6 @@ protected void onDestroy() {
         if (itemList != null) {
             itemList.clear();
         }
-        if (itemMap != null) {
```

Comment:
This is already done in `CollectionContainer` (it's also a field of `CollectionContainer`).